about/design-goals.md

```diff
@@ -0,0 +1,29 @@
+# CloudEvents Specification Design Goals
+
+CloudEvents are typically used in a distributed system to allow for services to be loosely coupled during development, then then configured with tight-coupling for specific applications.
+
+The goal of the CloudEvents specification is to allow for different cloud providers to create interoperable event systems, that allow for third-party tools and services to provide:
+- new event sources
+- new event consumers
+- observability, such as monitoring, debugging and visualization,
+- developer tools and libraries
+
+Key properties of the system:
```

Comment:
My first read of this made me think we needed to define these terms. I then read further and saw you mention the WG needs to define this vocabulary. :-) Not sure how or whether to rectify.

Reply:
Maybe its just me, but I suspect many people will never look at this doc - which is a shame because I think its very useful to help set the context for what the spec will talk about. Add to that, Mark's comment about needing to define the terms used in the following points, and my desire not to duplicate info, I'm wondering what people think about changing this PR so that it adds all of this info into the spec itself. This would serve a couple of purposes: 1) it will increase the changes of people reading it, 2) avoid duplication of very important concepts, 3) it'll make the spec more self-sufficient by beefing-up our intro (o where ever) section.

Reply:
If other folks agree, I could split this PR into two parts:
1. the goals/non-goals => spec
2. WG actions => README with link to Goals section

Reply:
I think splitting this up into the spec and goals is a great idea. Let me know if you'd like me to help with that.